### DISCUSSION TITLE
绝绝绝代代代肝肝肝，欢迎老板！

### DISCUSSION THREAD

**Discussion by mozhe0402** [share121/inter-knot#156] [其他]
![17232051157540f822a177b42f777b1192f78d90cf9821845](https://github.com/user-attachments/assets/9411aded-73fc-41bc-8fab-407fd5184c93)
开荒等级
1-20级 4r/1级
20-30级 7r/1级
30-40级 10r/1级
40-50级 15r/1级
主线一大章50
空洞花圃烈度11 20
三图探索全收集套餐（喵吉长官，宝箱遗失的
小卡格车，调查协会纪念）
套餐一：全收集=150r
套餐二：全收集+收集相关主线支线剧情=200r
基础代肝：日常/清体力（体力可以指定）
（包含咖啡店）80
精致包月代肝：日常/清体力/周常/零号空
洞/式舆防卫战/期间活动150
精品包月代肝：日常/清体力/周常/零号空洞/式舆防卫战/期间活动/角色精养/后续规划.280
只要是绝区零代肝啥都接 欢迎咨询
联系v:fttf6666fttf
ps，图是水社找的，有水印

**Comment by mozhe0402** [comment]
单机绳网？？？？？

**Comment by JinYi-qq** [comment]
能看得见

**Comment by caplise** [comment]
**不会旺季来？**
